wire-ios/Wire-iOS.xcodeproj/project.pbxproj

```diff
@@ -11127,7 +11138,7 @@
 				CODE_SIGN_IDENTITY = "iPhone Developer";
 				CODE_SIGN_STYLE = Manual;
 				COPY_PHASE_STRIP = NO;
-				CURRENT_PROJECT_VERSION = 1;
+				CURRENT_PROJECT_VERSION = 999999;
```

Comment:
should use the WIRE_BUILD_NUMBER  instead

Reply:
I don't understand yet, why there is a warning only for the `Wire Notification Service Extension` so I reverted the changes.